### DISCUSSION TITLE
Add outbound rule to Windows firewall

### DISCUSSION THREAD

**Discussion by LTusche** [rustdesk/rustdesk#5209] [Feature Request] [answered]
Hey there,
I installed the RustDesk Service on two systems so far and both times I couldn't connect before adding an outbound rule to the Windows firewall.
This wasn't an issue when using it as a pocket application, only after installing it as a service.

Could you add an outbound rule for the RustDesk.exe on installation, just to make sure this doesn't happen?

I really enjoy using it so far, thanks for your work 🔥

**Comment by LTusche** [comment]
Since no one else seems to have this issue I'm a bit unsure if the firewall is actually the problem.
I tried connecting a few times before installing and it worked without failing a single time.
Then after installing, I tried multiple times and it failed every time, even restarted the Pc to make sure the service auto starts.
I manually added an outbound rule allowing the RustDesk.exe to do whatever it wants and suddenly it worked every time again.

If this isn't a firewall issue, I have no idea what else was going wrong. If there is no answer yet, I will test again later this day.

**Comment by rustdesk** [comment]
Make sense, though I have never encountered problem

**Comment by rustdesk** [comment, marked as answer]
Just added https://github.com/rustdesk/rustdesk/commit/f47f2a91512feba30f0907f08690bbbae2311e75
